reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Variables.md

```diff
@@ -93,14 +93,53 @@ $Splat = @{ Name = "Win*"; Include = "WinRM" }
 Invoke-Command -Session $s -ScriptBlock { Get-Service @Using:Splat }
 ```
 
-## Using local variables in PowerShell 2.0
+### Other situations where the 'Using' scope modifier is needed
+
+For anything that executes out-of-runspace, doesn't execute directly in the
```

Comment:
Most users don't understand 'runspace', so I think we should use 'session'.  Something like:

'For any script or command that executes out of session, you need the `Using` scope modifier to embed variable values from the calling session scope,
so that out of session code can access them. The `Using` scope modifier is supported in the following contexts:'

Reply:
@PaulHigin, I agree;  just for context: the language comes from https://stackoverflow.com/a/60012780/45375, which @Jawz84 was kind enough to turn into this PR. In the SO answer, I was aiming for technical accuracy, but I do understand that the official documentation is (also) aimed at a less technical audience.